js/init.js

```diff
@@ -10,7 +11,7 @@ function cc_js_pageInit() {
 
 if (window.onload) {
     old_onload = window.onload;
-    window.onload = function () {
+    window.onload = () => {
```

Comment:
I don't know if there's any JS transpilation setup on this project, so I'd be hesitant to merge this and risk breaking on browsers that don't support this syntax. Thoughts @hugosolar ?

Reply:
Totally agree with you, we should care about accessibility. Also, we have a list of supported browsers and there's IE 11 which does not support [function shorthands](https://caniuse.com/#feat=arrow-functions)